packages/ibm-products/src/components/UserProfileImage/UserProfileImage.docs-page.js

```diff
@@ -1,3 +1,4 @@
+/* cSpell:disable */
```

Comment:
```suggestion
```

I would just disable the next line in this case too, but you could alternatively do ...

```/* cspell:words useravatar */```